target/linux/ipq806x/config-4.19

```diff
@@ -1,12 +1,19 @@
 CONFIG_ALIGNMENT_TRAP=y
```

Comment:
Missing commit description.

Reply:
About kernel config i don't really tell why 

Should i try to use the one from 4.14 ?

Reply:
Subject:  `ipq806x: update kernel config for 4.19`

> Why are these changes needed?

These changes are needed in order to migrate old kernel 4.14 config to the upcoming kernel version 4.19.

Reply:
Is it good now ?